### PR TITLE
New temperature report section

### PR DESCRIPTION
**Pull request by BobTorgerson**
This PR updates the temperature report section with a new annual min-mean-max table, along with a monthly min-mean-max table across 4 eras: historical, early century (2010-2039), mid century (2040-2069), and late century (2070-2099). This table has the ability to switch between all of the models and the scenarios to display different projected data.

It also has the new CSV preview along with pointing the CSV download link to the new monthly temperature data.

To run this, use the API branch: temperature_2km and set your env vars:
export SNAP_API_URL=http://localhost:5000
export RASDAMAN_URL=https://apollo.snap.uaf.edu/rasdaman/ows

### CODE REVIEW

File: components/reports/Temperature.vue [hunk 1]
```diff
@@ -14,199 +14,354 @@
...
+                  `${month}`
+                ].tasmean
+              }}<UnitWidget /><br />
+              <span class="small-text">
+                ({{
+                  results.temperature.summary.historical['CRU-TS'].historical[
+                    `${month}`
+                  ].tasmin
+                }}
+                &mdash;
```
Comment (cstephen) [inline components/reports/Temperature.vue:241]:
```suggestion
                &ndash;
```

**Review by cstephen — CHANGES_REQUESTED** [review]
This is looking great and is almost perfect, but a few minor suggestions:

- [x] Replace the `&mdash;` in the temperature ranges to an `&ndash;` (mentioned as a suggestion in the review comments).
- [ ] Increase the font size of the temperature ranges below the temperature mean value. These numbers are sorta hard to read currently. Hopefully they can be about the same size as the temperature ranges in the "Projected precipitation frequency" table.
- [x] As @brucecrevensten already mentioned in Slack, update `assets/mock.json` with the new temperature data structure so the app continues to work in mock API mode (i.e., `export MOCK_API=True`).

File: components/reports/Temperature.vue (hunk 1 above)
Comment (BobTorgerson) [reply components/reports/Temperature.vue]: I have changed them all to &ndash; now

**Comment by BobTorgerson** [comment]
This also fixes the unit conversions for all of the sections of the report, not just the temperature section.

Closes #336

**Pull request merged**